style.css

```diff
@@ -0,0 +1,535 @@
+  background-size: 50px 50px;
+}
+
+.speaker-img {
+  margin: 20px 0 0 20px;
+  height: 90px;
+  width: 90px;
+}
+
+.speaker-info {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  gap: 5px;
+}
+
+.speaker-name {
+  font-size: 15px;
+  color: var(--dark);
+}
+
+.speaker-description {
+  color: var(--orange);
+  font-style: italic;
+  font-size: 13px;
+  font-weight: 600;
+}
+
+.divisor-grey {
+  height: 1px;
+  width: 20px;
+  background-color: #d3d3d3;
+}
+
+.speaker-thematic {
+  color: rgb(75, 75, 75);
+  font-size: 11px;
+  font-weight: 400;
+}
+
+.see-more-btn {
+  font-size: 10px;
+  width: 100%;
+  padding: 10px;
+  max-width: 300px;
+  border-radius: 0;
+  border: 1px solid var(--grey);
+  background-color: transparent;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 8px;
+  margin-top: 40px;
+}
+
+.partners-section {
+  background-color: #414246;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 20px;
+  gap: 10px;
+}
+
+.partner-container {
+  display: flex;
+  width: 100%;
+  align-items: center;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap: 30px;
+}
+
+.partner-icon {
+  filter: saturate(0);
+  height: 50px;
+}
+
+.footer {
+  padding: 30px 0;
+  background-color: white;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 30px;
+}
+
+.logo-footer {
+  max-width: 40%;
+}
+
+.footer-text {
+  font-size: 10px;
+  color: rgb(100, 100, 100);
+}
+
+.section-title {
+  color: white;
+}
+
+/* -------RESPONSIVE------- */
+
+@media screen and (min-width: 768px) {
+  .show-desktop {
+    display: flex;
+  }
+
+  .hide-desktop {
+    display: none;
+  }
+
+  .home-hello {
+    font-size: 30px;
+    font-weight: 400;
+  }
+
+  .home-title {
+    font-size: 50px;
+  }
+
+  .home-description {
+    font-size: 16px;
+    max-width: 500px;
+  }
+
+  .cards-container {
+    justify-content: center;
+    flex-wrap: wrap;
+    flex-direction: row;
+    align-items: stretch;
+    gap: 25px;
+  }
+
+  .card {
+    flex-direction: column;
+    align-items: stretch;
+    width: auto;
+    max-width: 200px;
+    height: 200px;
+    justify-content: center;
+    gap: 10px;
+    padding: 20px 10px;
+  }
+
+  .card-icon {
+    width: 45px;
+    margin: auto;
+  }
+
+  .card-title {
+    width: 100%;
+    text-align: center;
+  }
+
+  .card-description {
+    font-size: 14px;
+    min-width: 100%;
+    text-align: center;
+  }
+
+  .speakers-container {
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+
+  .speakers-title {
+    font-size: 18px;
+    font-weight: 400;
+    color: var(--dark);
+  }
+
+  .speaker-card {
+    display: flex;
+    max-width: 45%;
+    justify-content: flex-start;
+    gap: 15px;
+    align-items: stretch;
+    background-image: url(./assets/icons/speaker-back.svg);
+    background-repeat: no-repeat;
+    background-size: 50px 50px;
+  }
+}
```

Comment:
- [ ] Kindly use the correct background design for each speaker. The background used for each speaker in the original template has a chessboard-like design while the one used for your design is that of a flower. Kindly make your design identical to the Behance design by implementing this change. 👍 😎 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/233157342-a779389b-1180-4ec7-99a6-35ec97ca6d71.png)

### Behance Design

![image](https://user-images.githubusercontent.com/77106962/233157479-4e7aeccc-684a-41bb-8e23-8edf51bf337e.png)